cli/src/main/scala/laserdisc/cli/CLI.scala

```diff
@@ -1,5 +1,4 @@
-package laserdisc
-package fs2
+package laserdisc.cli
```

Comment:
Minor thing, if you keep the package nested structure (i.e. as it was just changing `fs2` into `cli`) you can avoid the `import laserdisc._` below